sensors/sqs_sensor.py

```diff
@@ -142,15 +144,12 @@ def _may_setup_sqs(self):
             self.input_queues = []
 
         # checker configuration is update, or not
-        def _is_same_credentials(session, account_id):
-            if not session:
-                return False
-
+        def _is_same_credentials(session, account_id, credentials):
             c = session.get_credentials()
```

Comment:
I think it would be safer to leave the `if not session: return False` block. As it is you rely on session not being `None`, it's safer not to do that